src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const employeeList = [...document.querySelectorAll('tr')].slice(1, -1);
+const mainList = document.querySelector('tbody');
+const menu = document.querySelector('thead').firstElementChild.children;
+
+function salarySort(list) {
+  list.sort((a, b) => (+a.children[3].innerText.slice(1).split(',').join(''))
+   - (+b.children[3].innerText.slice(1).split(',').join('')));
+
+  mainList.append(...list);
+}
+
+function ageSort(list) {
+  list.sort((a, b) => (+a.children[2].innerText) - (+b.children[2].innerText));
+
+  mainList.append(...list);
+}
+
+function positionSort(list) {
+  list.sort((a, b) => (a.children[1].innerText
+    .localeCompare(b.children[1].innerText)));
+
+  mainList.append(...list);
+}
+
+function nameSort(list) {
+  list.sort((a, b) => (a.children[0].innerText
+    .localeCompare(b.children[0].innerText)));
+
+  mainList.append(...list);
+}
+
+menu[3].addEventListener('click', () => {
```

Comment:
use event delegation here
https://javascript.info/event-delegation

Reply:
Hi, Taras! Of course, you're right. Thank you for review. I have implemented all your recommendations.